catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/SolrCache.java

```diff
@@ -393,10 +393,12 @@ protected SolrQuery postAdapt(QueryRequest request, SolrFilterDelegate filterDel
                 SolrQuery query) throws UnsupportedQueryException {
             if (!request.isEnterprise()) {
                 List<SolrQuery> sourceQueries = new ArrayList<>();
-                for (String source : request.getSourceIds()) {
-                    sourceQueries.add(filterDelegate.propertyIsEqualTo(StringUtils.removeEnd(
-                            METACARD_SOURCE_NAME,
-                            SchemaFields.TEXT_SUFFIX), source, true));
+                if (request.getSourceIds() != null) {
```

Comment:
It'd be a little less cyclomatic complexity if you just updated line 394 to 
` if (!request.isEnterprise() && request.getSourceIds() != null) {`